AlfrescoApp/Model/Managers/SyncManager.m

```diff
@@ -268,6 +281,23 @@ - (AlfrescoNode *)alfrescoNodeForIdentifier:(NSString *)nodeId
 
 #pragma mark - Private Methods
 
+// this parses a path to get the relative path to the Sync folder
+- (NSString *) getRelativeSyncPath:(NSString *)oldPath
+{
+    NSString *newPath;
+    NSArray *array = [oldPath componentsSeparatedByString:@"Sync/"];
```

Comment:
Try to avoid hardcoded strings that may break if a seemingly unrelated area of code is changed. Specifically here, instead of `@"Sync"` this should use the `kSyncFolder` constant defined in `AlfrescoFileManager+Extensions.h`